DOCKER.md

```diff
@@ -0,0 +1,123 @@
+# Development with Docker
+
+## Features
+
+* Run development server requirements such as MySQL and Redis in containers.
+
+## Installing Docker
+
+Our Docker development environment requires at least Docker Compose version 2.23 or higher.
+These instructions will walk you through installing a compatible version.
+
+To install Docker, see the appropriate section below:
+[macOS](#macos), [Ubuntu](#ubuntu), [Windows](#windows)
+or consult your package management for your OS.
+
+### macOS
+
+1. Install Docker itself:
+```shell
+brew install docker
+```
+
+1. Install Docker Compose:
+    1. ```shell
+       export DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+       mkdir -p $DOCKER_CONFIG/cli-plugins
+       curl -SL https://github.com/docker/compose/releases/download/v2.27.0/docker-compose-darwin-$(uname -m) -o $DOCKER_CONFIG/cli-plugins/docker-compose
+       chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
+       ```
+    1. You can inspect that the version is appropriate by running `docker compose version` and inspecting the result.
+
+1. Install Colima as a container runtime:
+```shell
+brew install colima
+```
+
+1. Start the Colima service (and have it start on login)
+```shell
+brew services start colima
+```
+
+1. Verify that Docker works
+```shell
+docker run --rm hello-world
```

Comment:
I don't mind this `--rm` inclusion. It's generally true for the other `run` steps to automatically throw away the exited container.